src/utils/TriangleUtilities.js

```diff
@@ -86,11 +86,12 @@ export function getTriangleHitPointInfo( point, geometry, triangleIndex, target
 	// find the associated material index
 	let materialIndex = 0;
 	const groups = geometry.groups;
+	const firstVertexIndex = triangleIndex * 3;
 	for ( let i = 0, l = groups.length; i < l; i ++ ) {
 
 		const group = groups[ i ];
 		const { offset, count } = group;
```

Comment:
The const ```offset``` should really be named ```start```, which is the member of the group.

Reply:
Oh thanks for the catch! 😬 I must have forgotten to commit something because I swear I'd fixed these!